### PR TITLE
Fix issue QueueCommand and AddCommand cannot be found error 

### LINKED ISSUE #77: The functions QueueCommand and AddCommand cannot be found error
After migration to Hangfire 1.8.6 tasks stopped running getting this error below. We use Hangfire.Tags 1.8.5

System.ArgumentException: The functions QueueCommand and AddCommand cannot be found.
   at Hangfire.Tags.SqlServer.SqlTagsTransaction..ctor(SqlServerStorageOptions options, IWriteOnlyTransaction transaction)
   at Hangfire.Tags.SqlServer.SqlTagsServiceStorage.GetTransaction(IWriteOnlyTransaction transaction)
   at Hangfire.Tags.Storage.TagExpirationTransaction.Persist(String jobid)
   at Hangfire.Tags.States.TagsCleanupStateFilter.OnStateApplied(ApplyStateContext context, IWriteOnlyTransaction transaction)
   at Hangfire.States.StateMachine.InvokeOnStateApplied(Tuple`2 x)

### PR DESCRIPTION
**Pull request by 1bberto**
With the new `Hangfire.SqlServer 1.8.6` the method `AddCommand` from the class `SqlServerWriteOnlyTransaction`, was changed to `static` https://github.com/HangfireIO/Hangfire/commit/c0d6290153abca71e08834f3133222e0c110063e.

Updating to get the method with the proper `BindingFlags`, adding `BindingFlags.Static`

closes #77

### CODE REVIEW

**Comment by 1bberto** [comment]
@r-win, could you please take a look at this Pull Request, there is an error happening with the new Hangfire.SqlServer version,
thank you

**Comment by erwin-faceit** [comment]
I think this PR is not needed, you need to update to the preview of Hangfire.Tags, which is renamed to FaceIT.Hangfire.Tags.

**Comment by erwin-faceit** [comment]
It actually might be that this patch is also required on the hangfire_182 branch, but not on the master branch, because the master branch does NOT support hangfire 1.8+.

In that case, I would prefer it to leave the Hangfire.SqlServer reference alone (to the lowest possible), and use both BindingFlags so that the library remains backwards compatible.

**Comment by 1bberto** [comment]
Tks @erwin-faceit 
 I will make the change and the target branch

File: src/FaceIT.Hangfire.Tags.SqlServer/SqlTagsTransaction.cs [hunk 1]
```diff
@@ -57,7 +57,7 @@ public SqlTagsTransaction(SqlServerStorageOptions options, IWriteOnlyTransaction
             if (_queueCommand == null && _addCommand == null)
             {
                 _addCommand = transaction.GetType().GetTypeInfo().GetMethod(nameof(AddCommand),
-                    BindingFlags.NonPublic | BindingFlags.Instance);
+                    BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.Static);
```
Comment (1bberto) [inline src/FaceIT.Hangfire.Tags.SqlServer/SqlTagsTransaction.cs:60]: Add this to work with `Hangfire.SqlServer` > 1.8.6

**Comment by 1bberto** [comment]
@erwin-faceit / @r-win  , I've updated the code, moreover I added the other PR to update the branch `hangfire_182` #81, could you please take a look on both?

**Comment by alvicode** [comment]
we also need this.
Thanks a lot.

**Comment by erwin-faceit** [comment]
This PR will not be merged, since this branch does NOT support Hangfire 1.8+. Thank you for creating the PR in the hangfire_182 branch as well, that works as expected.

@alvicode: I think you're using the FaceIT.Hangfire.Tags packages, which already have this PR included. If you're still using the Hangfire.Tags packages: they are obsolete. I'll try to push stable versions of FaceIT.Hangfire.Tags packages somewhere this week.

**Comment by vjaiswal-mdsol** [comment]
Is this issue still persists, Because This PR is not merged

**Comment by erwin-faceit** [comment]
No. Do not use Hangfire.Tags with Hangfire >= 1.8.2. Switch to FaceIT.Hangfire.Tags instead.

**Pull request closed without merging**